install-spring-boot-on-an-oracle-instance.md

```diff
@@ -52,22 +52,22 @@ To successfully complete this tutorial, you must have the following:
 * An Oracle Cloud Infrastructure Free Tier account. [Start for Free]({{ site.urls.always_free }}).
 * A MacOS, Linux, or Windows computer with `ssh` support installed.
 
-## Set up a Compartment for Development
+## Set up a compartment for development
 
 Configure a compartment for your development.
 
-### Create a Compartment
+### Create a compartment
 
 Create a compartment for the resources that you create in this tutorial.
 
 1. Log in to the Oracle Cloud Infrastructure **Console**.
-2. Open the navigation menu and click **Identity & Security**. Under **Identity**, click **Compartments**.
-3. Click **Create Compartment**.
+2. Open the navigation menu and select **Identity & Security**. Under **Identity**, select **Compartments**.
+3. Select **Create Compartment**.
 4. Fill in the following information:
     * **Name:** `<your-compartment-name>`
     * **Description:** `Compartment for <your-description>.`
     * **Parent Compartment:** `<your-tenancy>(root)`
-5. Click **Create Compartment**.
+5. Select **Create Compartment**.
```

Comment:
If this isn't already in our style guide feel free to contribute an addition. We see a lot of "click," "press," "choose," and "select" and coming up with a nomenclature we can stick with would be a good idea.

Reply:
I've gravitated towards something akin to 'select' in the past just so it's a little more general and can include input from mobile devices.